src/libs/ajax/teaspoons/Teaspoons.ts

```diff
@@ -1,5 +1,6 @@
 import { jsonBody } from '@terra-ui-packages/data-client-core';
 import _ from 'lodash/fp';
+import * as qs from 'qs';
```

Comment:
OoC do we need to import `*` if we're only using the `stringify` method?

Reply:
great question. the `*` import is necessary for `qs` because that package doesn't export the individual members, it only exports everything as one single object (which has 3 functions in total: `stringify` and a couple of other related ones)

i was curious though and it turns out that the terra-ui typescript config allows the use of synthetic imports with `allowSyntheticDefaultImports: true`

so theoretically, `import { stringify } from 'qs';` should work fine.

but i think for the sake of maximum compatibility (i.e if compiler settings change), i'm going to keep it as-is. this is also consistent with the rest of the ajax classes in terra-ui which all import in the same way